src/Hydra/Serializer/DocumentationNormalizer.php

```diff
@@ -49,8 +52,9 @@ final class DocumentationNormalizer implements NormalizerInterface, CacheableSup
     private $urlGenerator;
     private $subresourceOperationFactory;
     private $nameConverter;
+    private $serializerContextFactory;
 
-    public function __construct(ResourceMetadataFactoryInterface $resourceMetadataFactory, PropertyNameCollectionFactoryInterface $propertyNameCollectionFactory, PropertyMetadataFactoryInterface $propertyMetadataFactory, ResourceClassResolverInterface $resourceClassResolver, OperationMethodResolverInterface $operationMethodResolver = null, UrlGeneratorInterface $urlGenerator, SubresourceOperationFactoryInterface $subresourceOperationFactory = null, NameConverterInterface $nameConverter = null)
+    public function __construct(ResourceMetadataFactoryInterface $resourceMetadataFactory, PropertyNameCollectionFactoryInterface $propertyNameCollectionFactory, PropertyMetadataFactoryInterface $propertyMetadataFactory, ResourceClassResolverInterface $resourceClassResolver, OperationMethodResolverInterface $operationMethodResolver = null, UrlGeneratorInterface $urlGenerator, SubresourceOperationFactoryInterface $subresourceOperationFactory = null, NameConverterInterface $nameConverter = null, ?SerializerContextFactoryInterface $serializerContextFactory = null)
```

Comment:
```suggestion
    public function __construct(ResourceMetadataFactoryInterface $resourceMetadataFactory, PropertyNameCollectionFactoryInterface $propertyNameCollectionFactory, PropertyMetadataFactoryInterface $propertyMetadataFactory, ResourceClassResolverInterface $resourceClassResolver, OperationMethodResolverInterface $operationMethodResolver = null, UrlGeneratorInterface $urlGenerator, SubresourceOperationFactoryInterface $subresourceOperationFactory = null, NameConverterInterface $nameConverter = null, SerializerContextFactoryInterface $serializerContextFactory = null)
```

Reply:
I made it nullable (and not just with a default `null` value, even if PHP make it nullable in this case) because a `null` value can be given, even if the BC was not handled.

Reply:
But it's not consistent with what we currently do in API Platform (and Symfony). When we'll change this, we'll change it everywhere in one batch using PHP CS Fixer.

Reply:
OK then. Too bad.